src/components/Navigation/Navigation.jsx

```diff
@@ -118,11 +123,30 @@ export default class Navigation extends React.Component {
         inputSelector: '.navigation__search-input'
       });
 
-      window.addEventListener('keyup', e => {
-        if (e.which === 9 && e.target.classList.contains('navigation__search-input')) {
-          this._openSearch();
+      // Keydown so we can "stop" the event if need be. 
+      window.addEventListener('keydown', e => {
+        // Short circuit, as we only care about the "naked" key
+        if (e.shiftKey || e.ctrlKey || e.metaKey) return;
```

Comment:
Not sure if this is really needed. Wouldn't the switch case event codes ignore it?

Reply:
yes you need those checks, please test in browser for experiment.